gmail_yaml_filters/upload.py

```diff
@@ -198,6 +199,7 @@ def upload_ruleset(ruleset, service=None, dry_run=False):
             request = service.users().settings().filters().create(userId='me', body=filter_data)
             if not dry_run:
                 request.execute()
+                time.sleep(1.5)
```

Comment:
When you are uploading/sending recipes into gmail and you want them to be in a specific order there needs to be a cooldown between sending messages. Gmail seems to process the recipe then add it to the list. If there is no delay then gmail could take a list of recipes designed to go 12345 and add it as 12435

It's only needed when you are sending the ruleset unsorted because you have rules designed on previous rules

I found ~1.5sec was the best